docker_daemon/README.md

```diff
@@ -9,7 +9,9 @@ Configure this Agent check to get metrics from docker_daemon service in real tim
 * Visualize and monitor docker_daemon states
 * Be notified about docker_daemon failovers and events.
 
-**Note**: The Docker check has been rewritten in Go for Agent v6 to take advantage of the new internal architecture. Hence it is still maintained but **only works with Agents <v6**. To learn how to use it with the Agent v6 [Consult our dedicated agent v6 setup](#agent-v6).
+**NOTE**: The Docker check has been rewritten in Go for Agent v6 to take advantage of the new internal architecture. Hence it is still maintained but **only works with Agents <v6**.
```

Comment:
since we're here, I'd change the `<v6` word in favor of something like `prior to major version 6` or something on the lines